index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Survey</title>
+</head>
+<body>
+    <div class="form">
+    <header>
+        <h1 class="title">freeCodeCamp Survey Form</h1>
+        <p class="description">Thank you for taking the time to help us improve the platform</p>
+    </header>
+    <form class="survey-form">
+        <div class="form-group">
+            <label class="name-label">Name</label></br>
+            <input type="text" class="name" name="Name" placeholder="Enter your name" required>
+        </div>
+        <div class="form-group">
+            <label class="email-label">Email</label></br>
+            <input type="text" class="email " name="Email" placeholder="Enter your Email" required>
+        </div>
+        <div class="form-group">
+            <label class="number-label">Age(optional)</label></br>
+            <input type="number" class="age" name="Age" min="10" max="99" placeholder="Age">
```

Comment:
- Try to avoid using the `<br>` tag instead use the CSS property for line space. And `<br>` tag is self enclosing tag, that's why you are getting a Stylelint error. You can read more about the `<br>` tag here [Mozilla](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br)